tests/drivers/clock_control/stm32_clock_configuration/boards/b_u585i_iot02a_pll_hsi_160.overlay

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2021 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*
+ * Warning: This overlay relies on initial b_u585i_iot02a configuration
+ * Any change to board configuration has impact on this file
+ */
+
+/delete-node/ &clk_lse;
+/delete-node/ &clk_msis;
+
+&clk_hsi {
+	status = "okay";
+};
+
+&pll1 {
+	div-m = <4>;
+	mul-n = <40>;
+	div-q = <2>;
+	div-r = <1>;
+	clocks = <&clk_hsi>;
+	status = "okay";
+};
```

Comment:
Don't you need to set:
```
&rcc {
	clock-frequency = <DT_FREQ_M(160)>;
```

What about redefining systematically SysClock source and not relying on board default ?
```
&rcc {
	clocks = <&XXXX>;
```